.bazelrc

```diff
@@ -322,7 +322,7 @@ build:plain-fuzzer --copt=-fsanitize=fuzzer-no-link
 build:plain-fuzzer --linkopt=-fsanitize=fuzzer-no-link
 
 build:asan-fuzzer --config=plain-fuzzer
-build:asan-fuzzer --config=asan
+build:asan-fuzzer --config=clang-asan
```

Comment:
Not sure if it's something wrong on my end, but I also needed to add (to make sure that clang-asan is using the right clang):
```
build:clang-asan --config=clang
```

Reply:
Should I leave this change in? I suppose I'd either revert the bazelrc changes or include this

Reply:
IMO we should add the new change (the `clang-asan` should include `--config=clang` or otherwise a different clang may be used).

Reply:
done!